### PR TITLE
handle error when submitting task

### LINKED ISSUE #1723: Show error message and retry when submitting tasks
https://github.com/LAION-AI/Open-Assistant/blob/4ab4821b05ba85a1d85bd6d94773cd0f0e5c452d/website/src/hooks/tasks/useGenericTaskAPI.tsx#L42

we should handle all kinds of errors instead of showing the NONE_AVAILABLE state. Maybe trigger a toast to let the user know or something.

### PR DESCRIPTION
**Pull request by notmd**
close #1723 

I also add loading state for submit and skip task

### CODE REVIEW

File: website/public/locales/en/error.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  "err_1003": "Invalid response type",
+  "err_1004": "Interaction request failed",
+  "err_1005": "Failed to generate task",
+  "err_1006": "No tasks of type {{task_type}} are currently available.",
+  "err_1007": "Task availability query failed",
+  "err_1008": "Message size too long",
+  "err_1009": "User recent messages have duplicates",
+  "err_1010": "Message text is empty",
+  "err_1011": "User cannot reply twice to the same message",
+  "err_1012": "User has too many pending tasks"
```
Comment (notmd) [inline website/public/locales/en/error.json:15]: I just copied these messages from the backend. Please suggest more friendly messages

Comment (notmd) [reply website/public/locales/en/error.json:15]: @andreaskoepf can you help on this?

Comment (AbdBarho) [reply website/public/locales/en/error.json:15]: we could add what the user could do, for example "Current  has too many pending tasks, please wait for a couple of minutes and try again."

**Review by AbdBarho — APPROVED** [review]
LGTM

**Comment by AbdBarho** [comment]
are there any retries here? as per https://github.com/LAION-AI/Open-Assistant/issues/1723

File: website/public/locales/en/error.json (hunk 1 above)
Comment (andreaskoepf) [reply website/public/locales/en/error.json:15]: I would show the error message that the backend sends in default case, e.g. I think this is better than "Something went wrong!". Regarding more explanatory error messages: Sure, could be done in a separate PR if someone is interested in analyzing when the error are generated and writing a more explanatory text, but IMO the strings as they are now are fine for v1.

**Comment by andreaskoepf** [comment]
Closes  #1138

**Comment by andreaskoepf** [comment]
> are there any automatic retries here? or do we just send the user back to editing state?

IMO we don't need automatic retry.. if the error is shown to the users and they can manually try to re-submit everything is good.

**Pull request merged**